Python/Python.sublime-build

```diff
@@ -8,8 +8,8 @@
 	"variants":
 	[
 		{
-			"name": "Syntax Check",
-			"shell_cmd": "python -m py_compile \"${file}\"",
```

Comment:
Why remove this variant?

Reply:
I don't see it's necessary.

Reply:
I don't have `pypy` installed, but I have Python 3.7, so I'm not convinced the standard "syntax check" command should be replaced with a less commonly available one...

Reply:
> I don't have pypy installed

I don't have it either, but it is the the most common interpreter after CPython, I would argue that we shouldn't have a `Ant` build system for Java.

> standard "syntax check"
How is it standard ?

Reply:
> I would argue that we shouldn't have a Ant build system for Java

if pypy is not worth having in the Python build system because it's a less common command, than why theres an `Ant` build system for java other than the standard `javac` build system (not everyone have Ant), I see it's the same case here, if I'm correct.

Reply:
> How is it standard?

it's using `python`, the same interpreter that is used to run/execute the file...

> if pypy is not worth having in the Python build system because it's a less common command

I don't think anybody said it's not worth having.

> than why theres an `Ant` build system for java other than the standard `javac` build system (not everyone have Ant), I see it's the same case here, if I'm correct.

No, because you are proposing to remove `python` (or to use your example, `javac`) and replace it with `pypy3` (`Ant` in your example), and disable the standard approach and only leave the non-standard one.

Reply:
No issues with adding another variant that seems useful.